config/test_cases/test_pull_request_openshift_ansible_tox.yml

```diff
@@ -18,6 +18,20 @@ actions:
     repository: "openshift-ansible"
     script: |-
       tox
+  - type: "script"
+    title: "build images for integration tests"
+    repository: "openshift-ansible"
+    script: |-
+      if [ -x test/integration/build-images.sh ]; then
```

Comment:
Will this fail as expected if the file doesn't exist? `-x` just checks for the user's execute bit, right?